### DISCUSSION TITLE
Dependency management

### DISCUSSION THREAD

**Discussion by rozsasarpi** [BAMresearch/probeye#52] [Ideas]
I just came across this tool that seems to manage dependencies as well: https://pyup.io/.

It looks like a better solution than #39, although it would be yet another external tool.

**Comment by joergfunger** [comment]
Isn't that something that "just" checks for security changes, not necessarily updates that are not due to a security problem?

**Reply by rozsasarpi** [reply]
I interpreted this:

![image](https://user-images.githubusercontent.com/6670894/143037834-91154c70-b1aa-4155-92ad-9d51f510f998.png)

in a way that it would check if there are more up to date dependencies (not only for security reasons) and even open a PR to upgrade to them. The PR would automatically trigger our pipeline.

I will try to find a clearer description what it can do.

**Reply by rozsasarpi** [reply]
[This](https://pyup.io/docs/bot/what-is-pyup-bot/) seems to confirm my interpretation.

Also there is this alternative to pyup: https://github.com/dependabot/dependabot-core.

**Comment by joergfunger** [comment]
Looks good
